docker-compose.yml

```diff
@@ -1,104 +1,147 @@
+      # auth config options
+      - AUTH_PORT=${AUTH_PORT:-1001}
+      - DONT_USE_KEYS=${DONT_USE_KEYS:-0}
+      - REWARDCODES=${REWARDCODES:-4,30}
+      # master config options
+      - MASTER_IP=${MASTER_IP:-localhost}
+      - MASTER_PORT=${MASTER_PORT:-2000}
+      - WORLD_PORT_START=${WORLD_PORT_START:-3000}
+      - USE_SUDO_AUTH=${USE_SUDO_AUTH:-0}
+      - USE_SUDO_CHAT=${USE_SUDO_CHAT:-0}
+      - USE_SUDO_WORLD=${USE_SUDO_WORLD:-0}
+      - PRESTART_SERVERS=${PRESTART_SERVERS:-1}
+      # world config options
+      - SOURCE=${SOURCE:-https://github.com/DarkflameUniverse/DarkflameServer}
+      - CHAT_SERVER_PORT=${CHAT_SERVER_PORT:-2005}
+      - DISABLE_CHAT=${DISABLE_CHAT:-0}
+      - PHYS_SPATIAL_PARTITIONING=${PHYS_SPATIAL_PARTITIONING:-1}
+      - PHYS_SP_TILESIZE=${PHYS_SP_TILESIZE:-102}
+      - PHYS_SP_TILECOUNT=${PHYS_SP_TILECOUNT:-24}
+      - DISABLE_EXTRA_BACKPACK=${DISABLE_EXTRA_BACKPACK:-0}
+      - DISABLE_VANITY=${DISABLE_VANITY:-0}
+      - SOLO_RACING=${SOLO_RACING:-0}
+      - DISABLE_ANTI_SPEEDHACK=${DISABLE_ANTI_SPEEDHACK:-0}
+      - CHECK_FDB=${CHECK_FDB:-0}
+      - CLASSIC_SURVIVAL_SCORING=${CLASSIC_SURVIVAL_SCORING:-0}
+      - PETS_TAKE_IMAGINATION=${PETS_TAKE_IMAGINATION:-1}
+      - DISABLE_DROPS=${DISABLE_DROPS:-0}
+      - HARDCORE_MODE=${HARDCORE_MODE:-0}
+      - HARDCORE_DROPINVENTORY_ON_DEATH=${HARDCORE_DROPINVENTORY_ON_DEATH:-1}
+      - HARDCORE_USCORE_ENEMIES_MULTIPLIER=${HARDCORE_USCORE_ENEMIES_MULTIPLIER:-2}
+      - HARDCORE_LOSE_USCORE_ON_DEATH_PERCENT=${HARDCORE_LOSE_USCORE_ON_DEATH_PERCENT:-10}
+      - ALLOW_NAMEPLATE_OFF=${ALLOW_NAMEPLATE_OFF:-0}
+      - LOG_IP_ADDRESSES_FOR_ANTI_CHEAT=${LOG_IP_ADDRESSES_FOR_ANTI_CHEAT:-1}
+      - HELP_0_SUMMARY=${HELP_0_SUMMARY:-Got an issue?}
+      - HELP_1_SUMMARY=${HELP_1_SUMMARY:-Stuck loading?}
+      - HELP_2_SUMMARY=${HELP_2_SUMMARY:-Missing features?}
+      - HELP_3_SUMMARY=${HELP_3_SUMMARY:-Get smashed?}
+      - HELP_4_SUMMARY=${HELP_4_SUMMARY:-Want to contribute?}
+      - HELP_0_DESCRIPTION=${HELP_0_DESCRIPTION:-Go to the DarkflameServer repository on GitHub to view issues and discussions about the server emulator!<br/><br/><a href="https://github.com/DarkflameUniverse/DarkflameServer"><font color="#0000EE">Click Here to go there!</font></a><br/><br/>}
+      - HELP_1_DESCRIPTION=${HELP_1_DESCRIPTION:-Try switching networks, using a VPN, or using your phone's hotspot to resolve the issue.<br/><br/>}
+      - HELP_2_DESCRIPTION=${HELP_2_DESCRIPTION:-While DarkflameServer is a mostly complete emulator, there are still some features that aren't implemented. You can track these on the GitHub issues page.<br/><br/>}
+      - HELP_3_DESCRIPTION=${HELP_3_DESCRIPTION:-Skill issue!<br/><br/>}
+      - HELP_4_DESCRIPTION=${HELP_4_DESCRIPTION:-Visit Discussions on the DarkflameServer GitHub page<br/>to ask questions and collaborate with other devs!<br/><br/>}
+      - ALLOW_PLAYERS_TO_SKIP_CINEMATICS=${ALLOW_PLAYERS_TO_SKIP_CINEMATICS:-0}
```

Comment:
I'm still not a fan of this huge copy paste here which is prone to errors, forgetting to update and is just messy overall.  Is there really no other possible way to do this more efficiently?

Reply:
Just going to remove most of them and leave only the necessary ones